runtime/filetype.vim

```diff
@@ -111,6 +111,9 @@ au BufNewFile,BufRead *.ino,*.pde		setf arduino
 " Ash of busybox
 au BufNewFile,BufRead .ash_history		setf sh
 
+" Asymptote
+au BufNewFile,BufRead *.asy		setf asy
```

Comment:
This should be `asymptote` now

Reply:
You mean `file.asymptote`? The common extension is `.asy`. I changed *the name* of the filetype which is different from the extension. E.g: filetype name of `.md` is markdown, and no one use `file.markdown` as a filename.

Reply:
`setf asymptote`